### PR TITLE
Update operator.yaml to bump up operands and move to latest ubi

### PR DESCRIPTION
**Pull request by rashmi43**
Fixes: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/48416

updated images from:
```
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:48a4bec3d1dec90b5dd5420bf7c41a5756b7fbe8b862546134fbe2caa607679f",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:ad1de47449386ae0f6009c38d714cbcc218d6f8087ea2d45272b671baa5072af",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:82b6decae91cf9cc9c3d65e264920e351850b2b3f4711b07d58d815fd4cdab3d",
         "platform": {
            "architecture": "ppc64le",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:3410b9e870bd51bd71646a85beb9e7dd767275208ff33f52c3277c1877c9a3b9",
         "platform": {
            "architecture": "s390x",
            "os": "linux"
         }
      }
   ]
}

### CODE REVIEW

File: build/Dockerfile.s390x [hunk 1]
```diff
@@ -16,11 +16,11 @@
 
 FROM alpine as builder
 
-RUN wget -O /qemu-s390x-static https://github.com/multiarch/qemu-user-static/releases/latest/download/qemu-s390x-static
+RUN wget -O /qemu-s390x-static https://github.com/multiarch/qemu-user-static/releases/v5.2.0-2/download/qemu-s390x-static
```
Comment (cwangVT) [inline build/Dockerfile.s390x:19]: I tested that the wget for the url is not working, the correct url should be: 
https://github.com/multiarch/qemu-user-static/releases/tag/v5.2.0-2/download/qemu-s390x-static ?

File: build/Dockerfile.ppc64le [hunk 2]
```diff
@@ -16,11 +16,11 @@
 
 FROM alpine as builder
 
-RUN wget -O /qemu-ppc64le-static https://github.com/multiarch/qemu-user-static/releases/latest/download/qemu-ppc64le-static
+RUN wget -O /qemu-ppc64le-static https://github.com/multiarch/qemu-user-static/releases/v5.2.0-2/download/qemu-ppc64le-static
```
Comment (cwangVT) [inline build/Dockerfile.ppc64le:19]: I tested that the wget for the url is not working, the correct url should be:
https://github.com/multiarch/qemu-user-static/releases/tag/v5.2.0-2/download/qemu-ppc64le-static ?

**Review by cwangVT — COMMENTED** [review]
pls verify the wget url

File: build/Dockerfile.ppc64le (hunk 2 above)
Comment (rashmi43) [reply build/Dockerfile.ppc64le]: yes, done I have updated it.

**Review by smagesh2106 — APPROVED** [review]
lgtm

**Pull request merged**